.travis.yml

```diff
@@ -1,31 +1,53 @@
+        recipients: rootpwa-push-notifications@lists.sourceforge.net
+        on_success: change
+        on_failure: always
+
 before_install:
     - sudo apt-get update -qq
-    - sudo apt-get install -y git dpkg-dev make g++ gcc binutils libx11-dev libxpm-dev libxft-dev libxext-dev
-    - sudo apt-get install -y gfortran libssl-dev libpcre3-dev xlibmesa-glu-dev libglew1.5-dev libftgl-dev libmysqlclient-dev libfftw3-dev cfitsio-dev graphviz-dev libavahi-compat-libdnssd-dev libldap2-dev python-dev libxml2-dev libkrb5-dev libgsl0-dev libqt4-dev
+    # packages required by ROOT
+    - sudo apt-get install -y git dpkg-dev make g++ gcc binutils libx11-dev libxpm-dev libxft-dev libxext-dev gfortran libssl-dev libpcre3-dev xlibmesa-glu-dev libglew1.5-dev libftgl-dev libmysqlclient-dev libfftw3-dev cfitsio-dev graphviz-dev libavahi-compat-libdnssd-dev libldap2-dev python-dev libxml2-dev libkrb5-dev libgsl0-dev libqt4-dev
+    # packages required by ROOTPWA
+    - sudo apt-get install -y libconfig++-dev libyaml-cpp-dev
+    - pip install --user pylint==1.5.5 numpy
+    - if [ ${TRAVIS_RPWA_USE_NLOPT} -eq 1 ] ; then sudo apt-get install -y libnlopt-dev ; fi
+    # fix Python library
+    - sudo rm -f /opt/python/2.7.10/lib/python2.7/config/libpython2.7.a
+    - sudo ln -sf /opt/python/2.7.10/lib/libpython2.7.so /opt/python/2.7.10/lib/python2.7/config/libpython2.7.so
+install:
     - ./travisCI/install-cmake.sh
-    - export PATH=$PWD/cmake/bin:${PATH}
+    - export PATH=${PWD}/cmake/bin:${PATH}
     - ./travisCI/install-boost.sh
-    - export BOOST_ROOT=$PWD/boost_1_58_0
-    - ./compileBoostLibraries.sh
-    - ./travisCI/install-libconfig.sh
-    - export LIBCONFIG=$PWD/libconfig_install
-    - export LD_LIBRARY_PATH=${LIBCONFIG}/lib:${LD_LIBRARY_PATH}
-    - ./travisCI/install-yaml-cpp.sh
-    - export YAML_CPP=$PWD/yaml-cpp-install
-    - export LD_LIBRARY_PATH=${YAML_CPP}/lib:${LD_LIBRARY_PATH}
+    - export BOOST_ROOT=${PWD}/boost-boost-1.61.0
     - ./travisCI/install-root.sh
     - . ./root/bin/thisroot.sh
-    - export ROOTPWA=$PWD
-install:
+    - if [ ${TRAVIS_RPWA_USE_BAT} -eq 1 ] ; then ./travisCI/install-bat.sh ; export BATINSTALLDIR=${PWD}/bat ; export PATH=${BATINSTALLDIR}/bin:${PATH} ; fi
+before_script:
+    # set some environment variables
+    - export ROOTPWA=${PWD}
+    - export LD_LIBRARY_PATH=${ROOTPWA}/build/lib:${LD_LIBRARY_PATH}
+    - export PATH=${ROOTPWA}/build/bin:${PATH}
+    - export PYTHONPATH=${ROOTPWA}/build/pyLib:${PYTHONPATH}
+script:
     - cd ${ROOTPWA}/build
     - cmake ..
-    - make
-before_script: echo "before_script nop"
-script: echo "script nop"
-cache: apt
-notifications:
-    email:
-        recipients: rootpwa-push-notifications@lists.sourceforge.net
-        on_success: change
-        on_failure: always
+    # run 'make'
+    # - if 'make' was not successful, fail the build
+    # - if 'make' was successful, run 'make test'
+    # - if 'make test' was not successful, print the log and fail the build
```

Comment:
I added this line; but have no idea why github does not understand this diff is outdated